api/src/routes/user.ts

```diff
@@ -1,12 +1,28 @@
+import _ from 'lodash';
+import { ObjectId } from 'mongodb';
 import { type FastifyPluginCallbackTypebox } from '@fastify/type-provider-typebox';
 
 import { schemas } from '../schemas';
+import {
+  type ProgressTimestamp,
+  getCalendar,
+  getPoints
+} from '../utils/progress';
+import { normalizeTwitter } from '../utils/normalize';
+import { encodeUserToken } from '../utils/user-token';
+
+const removeNulls = (obj: Record<string, unknown>) =>
+  _.pickBy(obj, value => value !== null);
```

Comment:
I was wondering what to do about the `null`s:
> // TODO: Handle `null` on server?

So, you think this should be handled by the server? We should not just send back `null` or `<value>`